deeppavlov/models/torch_bert/torch_bert_classifier.py

```diff
@@ -0,0 +1,199 @@
+            raise RuntimeError('Set return_probas to True for multilabel classification!')
+
+        super().__init__(optimizer=optimizer,
+                         optimizer_parameters=optimizer_parameters,
+                         **kwargs)
+
+    def train_on_batch(self, features: List[InputFeatures], y: Union[List[int], List[List[int]]]) -> Dict:
+        """Train model on given batch.
+        This method calls train_op using features and y (labels).
+
+        Args:
+            features: batch of InputFeatures
+            y: batch of labels (class id or one-hot encoding)
+
+        Returns:
+            dict with loss and learning_rate values
+        """
+        input_ids = [f.input_ids for f in features]
+        input_masks = [f.attention_mask for f in features]
+
+        b_input_ids = torch.cat(input_ids, dim=0).to(self.device)
+        b_input_masks = torch.cat(input_masks, dim=0).to(self.device)
+        b_labels = torch.from_numpy(np.array(y)).to(self.device)
+
+        self.optimizer.zero_grad()
+
+        loss, logits = self.model(b_input_ids, token_type_ids=None, attention_mask=b_input_masks,
+                                  labels=b_labels)
+        loss.backward()
+        # Clip the norm of the gradients to 1.0.
+        # This is to help prevent the "exploding gradients" problem.
+        if self.clip_norm:
+            torch.nn.utils.clip_grad_norm_(self.model.parameters(), self.clip_norm)
+
+        self.optimizer.step()
+        if self.lr_scheduler is not None:
+            self.lr_scheduler.step()
+
+        return {'loss': loss.item()}
+
+    def __call__(self, features: List[InputFeatures]) -> Union[List[int], List[List[float]]]:
+        """Make prediction for given features (texts).
+
+        Args:
+            features: batch of InputFeatures
+
+        Returns:
+            predicted classes or probabilities of each class
+
+        """
+        input_ids = [f.input_ids for f in features]
+        input_masks = [f.attention_mask for f in features]
+
+        b_input_ids = torch.cat(input_ids, dim=0).to(self.device)
+        b_input_masks = torch.cat(input_masks, dim=0).to(self.device)
+
+        with torch.no_grad():
+            # Forward pass, calculate logit predictions
+            logits = self.model(b_input_ids, token_type_ids=None, attention_mask=b_input_masks)
+            logits = logits[0]
+
+        if self.return_probas:
+            if not self.multilabel:
+                pred = torch.nn.functional.softmax(logits, dim=-1)
+            else:
+                pred = torch.nn.functional.sigmoid(logits)
+            pred = pred.detach().cpu().numpy()
+        else:
+            logits = logits.detach().cpu().numpy()
+            pred = np.argmax(logits, axis=1)
+        return pred
+
+    @overrides
+    def load(self, fname=None):
+        if fname is not None:
+            self.load_path = fname
+
+        if self.pretrained_bert and not os.path.isfile(self.pretrained_bert):
```

Comment:
in case likes that we can use `Path` from `pathlib` and remove `import os`

Reply:
if you want you can fix it. For it's not important.

Reply:
fixed